Shopist/ShopistUITests/ShopistUITests.swift

```diff
@@ -1,18 +1,57 @@
 /*
-* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
-* This product includes software developed at Datadog (https://www.datadoghq.com/).
-* Copyright 2019-2020 Datadog, Inc.
-*/
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
 
 import XCTest
 
 class ShopistUITests: XCTestCase {
-    override func setUp() {
+    override func setUp() { // swiftlint:disable:this empty_xctest_method
         continueAfterFailure = false
     }
 
-    func testAppCanBeLaunched() {
+    func testRunFullCheckoutFlow() {
+        let sessionCount = 3
         let app = XCUIApplication()
-        app.launch()
+        for sessionIndex in 0...sessionCount {
+            app.launch()
+
+            app.collectionViews.firstMatch.goToOffScreenCell("lastCell")?.tap()
+            app.collectionViews.firstMatch.goToOffScreenCell("lastCell")?.tap()
+
+            let navBar = app.navigationBars.firstMatch
+            navBar.buttons["cart.badge.plus"].tap()
+            navBar.buttons["cart"].tap()
+
+            let cardButton = app.navigationBars["Cart"].buttons["creditcard"]
+            cardButton.tap()
+            if !app.alerts.firstMatch.waitForExistence(timeout: 5.0) {
+                fatalError("Alert should show up!")
+            }
+            app.alerts.firstMatch.buttons.firstMatch.tap()
+
+            if sessionIndex == sessionCount {
+                Thread.sleep(forTimeInterval: 5.0)
+            }
+            app.terminate()
+        }
+    }
+}
+
+private extension XCUIElement {
+    func goToOffScreenCell(_ identifier: String) -> XCUIElement? {
```

Comment:
nit: would be more elegant to call it `findCell(identifier: String)`, because we're using it with `.tap()`:
```swift
app.collectionViews.firstMatch.findCell(identifier: "lastCell")?.tap()
```
The "go to off screen cell" refers to the implementation, which may not always be true.